### DISCUSSION TITLE
Announce a vacancy for a Meteor.js internship?

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#165] [Ideas]
We could post a vacancy on our website, for instance 6 months internship, paid 500 euros per month, for a Meteor.js intern. Once it's on the site we could post it on various job boards and see who sends us their CV! :)

Maybe that would work better than the Twitter/mailing list messages we've tried so far?

**Comment by hugoroy** [comment]
It would require an intern manager. @piks3l what do you think?

**Comment by evilcel3ri** [comment]
To give q bit of context, this idea came out after our discussion with Craig from the Web Foundation, there was a possibility they can help us financially speaking but nothing sure so far. We discussed it with @michielbdejong and it seemed as a good idea but I haven't thought it through yet.

To manage the intern, yes of course that is something that need to be present. Nonetheless who? Michiel would be good for the technical issues as I don't know much about Meteor yet (I intend to learn). To be honest, the only thing I know is that the edit.tosdr.org tool needs to be developed further and an intern would be a good opportunity for that. Our team could managed it and I can, if necessary, ensure a daily presence (but how do we count that?). 

I'm really interested to hear your viewpoints.

**Comment by hugoroy** [comment]
Thanks for the context. If the funding and contract were actually handled by the Web Foundation, that would make things *way* easier! My guess is also that Michiel would be a better fit as it's a technical internship

**Comment by michielbdejong** [comment]
We probably don't have time to help an intern who wants to *learn* meteor, but we can review their pull requests. I agree we need to offer them an office to work at, otherwise it's not much of an "internship". I could also ask people at FabLab Berlin if they could spare a desk to house an intern for us. Let's see if we can organize this.
